components/RegisterPatient.tsx

```diff
@@ -94,10 +94,41 @@ export function RegisterPatient() {
       providerId,
     } = values;
 
+    const {data: existingPatients, error} = await supabase
+      .from('Patients')
+      .select('*')
+      .eq('first_name', firstName.toLowerCase())
+      .eq('surname', surname.toLowerCase())
+      .eq('date_of_birth', dateOfBirth);
+      console.log("Existing Patients:", existingPatients);
+
```

Comment:
This still allows me to create a user. I think you may have to use the `match` filter instead of `eq` https://supabase.com/docs/reference/javascript/match.

I think we may have to pay attention to the case. Do we need to query a case insensitive value, or make it case insensitive and compare the results with the submitted values?